dex/testing/dcrdex/harness.sh

```diff
@@ -155,7 +155,7 @@ if [ $ZEC_ON -eq 0 ]; then
         {
             "base": "ZEC_simnet",
             "quote": "BTC_simnet",
-            "lotSize": 1000000,
+            "lotSize": 1000000000,
```

Comment:
ZEC lot size of 0.01 was really low, and I got a dust output redeeming BTC with a price of 0.01.